pkg/controller/storagecluster/reconcile.go

```diff
@@ -95,6 +96,9 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 	// If Update request is made and StorageCluster is PhaseIgnored, no need to
 	// proceed further
 	if instance.Status.Phase == "" {
+		if instance.Spec.Version == "" {
+			instance.Spec.Version = version.Version
```

Comment:
Hi, why is it under `Spec` and not under `Status`?

Reply:
@guymguym , according to the following doc, status is the current state of the object and specification is the complete configuration of the desired state...

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status

Couple of projects which I referred also had the same patern,
https://github.com/nats-io/nats-operator/blob/master/pkg/apis/nats/v1alpha2/cluster.go
https://github.com/libopenstorage/operator/blob/master/pkg/apis/core/v1alpha1/storagecluster.go